README.md

```diff
@@ -193,19 +194,31 @@ For most users, setting `image: [path-to-image]` on a per-page basis should be e
 * `path` - The relative path to the image. Same as `image: [path-to-image]`
 * `twitter` - The relative path to a Twitter-specific image.
 * `facebook` - The relative path to a Facebook-specific image.
-* `height` - The height of the Facebook (`og:image`) image
-* `width` - The width of the Facebook (`og:image`) image
+* `height` - The height of the Facebook (`og:image`) image and JSON-LD image object.
+* `width` - The width of the Facebook (`og:image`) image and JSON-LD image object.
+
+The JSON-LD will default to the `image: ` tag unless `path: ` `height: ` and `width: `.
 
 You can use any of the above, optional properties, like so:
 
 ```yml
 image:
+  path: /img/banner.png
   twitter: /img/twitter.png
   facebook: /img/facebook.png
   height: 100
   width: 100
 ```
 
+Or if you have no site specific images simply:
```

Comment:
The use of "site" here is confusing. Perhaps "service", "network" or explicitly call out Facebook and Twitter?

I'd also have the simpler and more common use case first and let users opt into additional complexity as they need it, not the other way around.